data/pages.json

```diff
@@ -18,6 +18,11 @@
         "imageUrl": "https://raw.githubusercontent.com/tsparticles/404-templates/main/__screenshots/space.png?raw=true",
         "url": "space/404.html"
     },
+    {
```

Comment:
Please append your theme at the end of the file

Reply:
My bad, I went alphabetically